Unix/regress

```diff
@@ -278,7 +278,7 @@ fi
 ##
 
 rm -rf $prefix
-./configure --prefix=$prefix --outputdirname=$current_output $options
+./configure --enable-werror --prefix=$prefix --outputdirname=$current_output $options
```

Comment:
I see no harm in enabling it for Microsoft builds. We have fixed the necessary issues now, so lets keep it clean.